api/session/auth_session.py

```diff
@@ -0,0 +1,79 @@
+from datetime import datetime
+
+import jwt
+from api.session.exceptions import SessionCreateError
+from config.env import env
+from flask import abort
+from flask import make_response
+from flask import redirect
+from flask import request
+from flask import session
+from flask.views import MethodView
+from security.utils import create_token
+from security.utils import validate_token
+
+
+class AuthSessionView(MethodView):
+    """
+    Views for session related operations
+    """
+
+    @staticmethod
+    def user():
+        """
+        GET /sessions/user endpoint
+        Shows the user details of the current user session
+        or an error if no authenticated user session found
+        :return: 200 user details json or 404 error
+        """
+        token = request.cookies.get(
+            env.config.get("FSD_USER_TOKEN_COOKIE_NAME")
+        )
+        if token:
+            try:
+                valid_token = validate_token(token)
+                return make_response(valid_token), 200
+            except jwt.PyJWTError as e:
+                # abort(404, "Session token expired or invalid")
```

Comment:
Can we remove this comment? Did you want this text in the response below?

Reply:
Yep thanks for spotting. Updated to formatted error_response